api/v1beta1/network_disruption_test.go

```diff
@@ -442,6 +442,22 @@ var _ = Describe("NetworkDisruptionSpec", func() {
 			})
 		})
 
+		Describe("test option limits", func() {
+			It("rejects bandwidthLimits below 32 bytes", func() {
```

Comment:
These tests are working, but In this kind of  scenario you can use the `DescribeTable` 😉